core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -228,7 +236,7 @@ class KafkaApis(val requestChannel: RequestChannel,
     val partitionsRemaining = controller.shutdownBroker(controlledShutdownRequest.brokerId)
     val controlledShutdownResponse = new ControlledShutdownResponse(controlledShutdownRequest.correlationId,
       Errors.NONE, partitionsRemaining)
-    requestChannel.sendResponse(new Response(request, new RequestOrResponseSend(request.connectionId, controlledShutdownResponse)))
+    sendResponseExemptThrottle(request, new Response(request, new RequestOrResponseSend(request.connectionId, controlledShutdownResponse)))
```

Comment:
In this case `authorizeClusterAction` method throws an exception and the response is sent from the exception handler in `handle`. All responses sent from the exception handler are throttled.